fb-negotiation/core/controller.py

```diff
@@ -0,0 +1,70 @@
+from cocoa.core.controller import Controller as BaseController
+
+class Controller(BaseController):
+    def __init__(self, scenario, sessions, chat_id=None, debug=False):
+        super(Controller, self).__init__(scenario, sessions, chat_id, debug)
+        self.marked_agree = [False, False]
+        self.quit = False
+        self.outcomes = [None, None]
+        # self.prices = [None, None]
+
+    def event_callback(self, event):
+        if event.action == 'accept':
```

Comment:
What are the actions in the fb setting? I don't think they have explicit accept or reject, instead two people select the objects they want. Can we confirm this?